src/style.css

```diff
@@ -1 +1,38 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  background-image: none;
+}
```

Comment:
Avoid using the `none` value for `background-image` unless it's necessary to override a previously set value. If the element doesn't have a background image by default, this declaration can be removed.